pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -776,6 +781,20 @@ func (z *zedrouter) initSubscriptions() (err error) {
 	if err != nil {
 		return err
 	}
+
+	// Information about patch envelopes
+	z.subPatchEnvelopeInfo, err = z.pubSub.NewSubscription(pubsub.SubscriptionOptions{
+		AgentName:   "zedagent",
+		MyAgentName: agentName,
+		TopicImpl:   types.PatchEnvelopes{},
+		Activate:    false,
```

Comment:
Isn't it necessary to activate the subscription eventually?

Reply:
Done